slack_sdk/socket_mode/async_client.py

```diff
@@ -46,8 +46,17 @@ async def issue_new_wss_url(self) -> str:
             )
             return response["url"]
         except SlackApiError as e:
-            self.logger.error(f"Failed to retrieve Socket Mode URL: {e}")
-            raise e
+            if e.response["error"] == "ratelimited":
+                # NOTE: ratelimited errors rarely occur with this endpoint
```

Comment:
As long as you don't consume the WSS URL, excessive `apps.connections.open` API calls do not result in rate-limited errors.